modules/core/src/main/java/org/mycontroller/standalone/message/McMessage.java

```diff
@@ -54,6 +54,7 @@
     @NonNull
     private Integer gatewayId;
     private String nodeEui;
+    private String name;
```

Comment:
@Fraid We cannot add name here. MySensors and MyController network won't support directly. Hence we are following a different approach to update name.

We have to deal it on `PhilipsHueProviderBridge.java`,[ like here](https://github.com/mycontroller-org/mycontroller/blob/development/modules/core/src/main/java/org/mycontroller/standalone/provider/rflink/RFLinkProviderBridge.java)